lib/api/request_adapter.rb

```diff
@@ -58,6 +58,8 @@ def json_body
       @json_body ||= begin
                        body = @request.body.read if @request.body
                        body.blank? || body == "null" ? {} : JSON.parse(body)
+                     rescue JSON::ParserError
+                       @request.params
```

Comment:
This might turn out to be necessary, but curious that in the POST case the body comes in as JSON but in the OPTIONS case it is key=val seems like it might be something else causing the body not to be coming in as JSON

Reply:
think options are not expecting parameters and that is why this is getting messed up